### DISCUSSION TITLE
Is rendering browser dependent?

### DISCUSSION THREAD

**Discussion by refjord** [niklasvh/html2canvas#3042] [Q&A]
Hello,

I think I've not found sufficient information on this topic (or maybe skipped it?).

Anyways I wonder if rending with html2canvas is browser specific, meaning there can be differences in the resulting image depending on the browser that is used.

Or is the library using an internal rending engine and the images look the same in every supported browser?

I think the latter is the case but I'm not sure as I don't really know what to test for.

Thanks!

**Comment by codeHunterChan** [comment]
there're some problems on firefox
